OrbitBase/ThreadUtilsLinux.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <OrbitBase/Logging.h>
+#include <sys/syscall.h>
+#include <time.h>
+#include <unistd.h>
+
+#include "OrbitBase/ThreadUtils.h"
+
+namespace orbit_base {
+
+[[nodiscard]] uint32_t GetCurrentThreadId() {
+  thread_local pid_t current_tid = syscall(__NR_gettid);
+  return static_cast<uint32_t>(current_tid);
+}
+
+std::string GetThreadName(uint32_t tid) {
```

Comment:
Same here, I don't like this is unsigned when on Linux pids are signed and we use `int32_t` in the codebase for pids.